.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: python
 python:
-- '2.7'
```

Comment:
Too soon? do you have any problem with the 2.7?

Reply:
Yes, there is an issue in CI with py27. I'll take a look today

Reply:
Fixed